schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -374,38 +377,45 @@ private SchemaString schemaStringFromJson(String json) throws IOException {
     }
 
     public static class SchemaVersionEntry {
-        private String name;
+        private String subject;
         private Integer version;
+        private Long id;
         private String schema;
 
         public SchemaVersionEntry() {
         }
 
-        public SchemaVersionEntry(String name, Integer version, String schema) {
-            this.name = name;
+        public SchemaVersionEntry(String subject, Integer version,Long id, String schema) {
+            this.subject = subject;
```

Comment:
This object can be removed, there is Schema object already matching this and confluents, simply methods should create that and return it.

Reply:
@michaelandrepearce Makes sense -- have changes ready and will push them